app/controllers/home_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
+    @is_sign_in = user_signed_in?
```

Comment:
why you need `@is_sign_in` variable here?

Reply:
It's my mistake, this variable is not used, I have remove it at newest commit